tests/bootstrap.php

```diff
@@ -54,6 +55,14 @@
     ],
 ]);
 
+Cache::setConfig([
+    '_cake_translations_' => [
+        'engine' => 'File',
+        'prefix' => 'cake_core_',
+        'serialize' => true,
```

Comment:
```suggestion
        'engine' => 'Array',
```

No need to write to files during tests.

Reply:
Then we should do that in the [core testsuite](https://github.com/cakephp/cakephp/blob/5.x/tests/bootstrap.php#L92-L103) as well